dashboard/src/pages/Hardware/HardwareTable.tsx

```diff
@@ -0,0 +1,244 @@
+import type {
+  ColumnDef,
+  ColumnFiltersState,
+  PaginationState,
+  Row,
+  SortingState,
+} from '@tanstack/react-table';
+
+import {
+  flexRender,
+  getCoreRowModel,
+  getFilteredRowModel,
+  getPaginationRowModel,
+  getSortedRowModel,
+  useReactTable,
+} from '@tanstack/react-table';
+
+import { memo, useCallback, useMemo, useState } from 'react';
+
+import { FormattedMessage } from 'react-intl';
+
+import type { LinkProps } from '@tanstack/react-router';
+
+import BaseTable, { TableHead } from '@/components/Table/BaseTable';
+
+import { formattedBreakLineValue } from '@/locales/messages';
+
+import {
+  TableBody,
+  TableCell,
+  TableCellWithLink,
+  TableRow,
+} from '@/components/ui/table';
+
+import { BuildStatus, GroupedTestStatus } from '@/components/Status/Status';
+import { TableHeader } from '@/components/Table/TableHeader';
+import { PaginationInfo } from '@/components/Table/PaginationInfo';
+
+import type { HardwareTableItem } from '@/types/hardware';
+
+import { sumStatus } from '@/utils/status';
+
+import { InputTime } from './InputTime';
+
+const columns: ColumnDef<HardwareTableItem>[] = [
+  {
+    accessorKey: 'hardwareName',
+    header: ({ column }): JSX.Element =>
+      TableHeader({
+        column: column,
+        sortable: true,
+        intlKey: 'global.name',
+        intlDefaultMessage: 'Tree',
```

Comment:
imo this should be "Hardware" instead of name, but either way the default message is wrong here
```suggestion
        intlKey: 'global.hardware',
        intlDefaultMessage: 'Hardware',
```